assets/js/components/NewChecksResults/ClusterChecksResultsPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getCatalog } from '@state/selectors/catalog';
+import { getLastExecution } from '@state/selectors/lastExecutions';
+import { getCluster, getClusterHostNames } from '@state/selectors/cluster';
+import { updateCatalog } from '@state/actions/catalog';
+import { updateLastExecution } from '@state/actions/lastExecutions';
+import { RUNNING_EXECUTION_STATE } from '@state/lastExecutions';
+import ExecutionHeader from './ExecutionHeader';
+import ExecutionResults from './ExecutionResults';
+import { getCheckResults } from '../ExecutionResults/checksUtils';
+
+function ClusterChecksResultsPage() {
```

Comment:
Why not `ExecutionResultsPage` as it was before?
We are bouncing around this name all the time hehe

Reply:
Having removed the `New` thingy this does not apply anymore.

However something to take into account as we evolve is that checks results would be scoped byCluster, bySAPSystem, byWhatever. But let's see how that evolves, for now we should be good.